### ISSUE TITLE
Add model: Stable Audio Open 1.0

### ISSUE THREAD

**Issue by rahuldshetty** [PABannier/bark.cpp#180]
Stability AI has open-sourced their Audio/Sound generation model: [Stable Audio Open 1.0](https://huggingface.co/stabilityai/stable-audio-open-1.0). This would be a great addition to this library. 

Source Code: https://github.com/Stability-AI/stable-audio-tools

**Comment by PABannier** [comment]
This is super interesting indeed! Putting it high in my priority list :)

**Comment by Amin456789** [comment]
yes yes yes! thank u for ur hard work, i would suggest to make a new page like stable audio.cpp when u r about to making it

**Comment by lin72h** [comment]
It would be very useful for sampling artist

**Comment by olilarkin** [comment]
yes please!
